package.json

```diff
@@ -138,7 +138,7 @@
     "babel-register": "^6.6.0",
     "babel-types": "^6.6.4",
     "babylon": "^6.6.4",
-    "base64-js": "^0.0.8",
+    "base64-js": "^1.0.4",
```

Comment:
Why not `^1.1.0` as that version seems to have a bunch of improvements over `1.0.4` (whereas `1.1.1` and `1.1.2` seem to only be about aesthetics).